backend/workers/google/src/docs/create_document.py

```diff
@@ -36,7 +36,6 @@ def create_document():
             params = Struct()
             params.update({
                 "workflowStepId": args["workflowStepId"],
-                "docTitle": document.get('title')
```

Comment:
why did you remove this?

Reply:
Data wasn't relevant for the user, I think we shouldn't expose it unless we could provide a link for it or something usable for the final user